src/PrestaShopBundle/Command/ModuleCommand.php

```diff
@@ -183,9 +195,15 @@ protected function executeConfigureModuleAction($moduleName, $file = null)
         );
     }
 
-    protected function executeGenericModuleAction($action, $moduleName)
+    protected function executeGenericModuleAction($action, $moduleName, $options = null)
```

Comment:
```suggestion
    protected function executeGenericModuleAction($action, $moduleName, array $options = null)
```

Reply:
We can't do that, as the check is make to see if we have an array to pass it to the action.